ONSdigital/dp-frontend-dataset-controller
# Feature/implement controller middleware

### What

- Updated go dependencies
- Updated handlers to implement the `ControllerHandlers` middleware and updated methods accordingly
- Updated zebedee client interface methods parameter lists to align with the method implementations in the `dp-api-clients-go` zebedee package
- Updated mappers to accept `lang` to map to page model
- Updated tests and regenerated mocks

### How to review

- Ensure tests pass and code builds.
- Ensure changes make sense and unnecessary code is removed from handlers where applicable. Worthwhile checking if I've missed any places
- Ensure pages handled by frontend dataset controller load 

### Who can review

Anyone
